Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -0,0 +1,1129 @@
+            foreach (QtestTestStep tsStep in tsTest.Steps)
+            {
+                QtestTestStep newtsStep = new QtestTestStep();
+                newtsStep.StepID = tsStep.StepID.ToString();
+                newtsStep.StepName = tsStep.StepName;
+                newtsStep.Description = tsStep.Description;
+                newtsStep.Expected = tsStep.Expected;
+                newTSTest.Steps.Add(newtsStep);
+            }
+
+            ////Get the TC parameters and their selected value
+            //if (linkedTest != null)
+            //{
+            //    if ((linkedTest.Split(';')[0].ToString() != tsTest.Name.ToString()) || (linkedTest.Split(';')[0].ToString() != tsTest.TestName.ToString()))
+            //    {
+            //        if (newTSTest.Description == null)
+            //        {
+            //            newTSTest.Description = string.Empty;
+            //        }
+            //        newTSTest.Description = tsTest.TestName.ToString() + System.Environment.NewLine + newTSTest.Description;
+            //    }
+            //}
+            //else
+            //{
+            //    //Regular TC
+            //    for (int i = 0; i <= tsTest.Params.Count - 1; i++)
+            //    {
+            //        QtestTestParameter newtsVar = new QtestTestParameter();
+            //        if (tsTest.Parameters.na(i) != null) { newtsVar.Name = tsTest.Params.ParamName(i); }
+            //        if (tsTest.Params.ParamValue(i) != null) { newtsVar.Value = tsTest.Params.ParamValue(i).ToString(); }
+            //        if (tsTest.Params.Type(i) != null) { newtsVar.Type = tsTest.Params.Type(i).ToString(); }
+            //        newTSTest.Parameters.Add(newtsVar);
+            //    }                
+            //}
+
+            //Get the TC execution history
+            try
+            {
+                List<QtestTestRun> TSTestRuns = GetListTSTestRuns(tsTest);
+                newTSTest.Runs = new List<QtestTestRun>();
+                foreach (Run run in TSTestRuns)
+                {
+                    QtestTestRun newtsRun = new QtestTestRun();
+                    newtsRun.RunID = run.ID.ToString();
+                    newtsRun.RunName = run.Name;
+                    newtsRun.Status = run.Status;
+                    newtsRun.ExecutionDate = (run["RN_EXECUTION_DATE"]).ToString();
+                    newtsRun.ExecutionTime = (run["RN_EXECUTION_TIME"]).ToString();
+                    newtsRun.Tester = (run["RN_TESTER_NAME"]).ToString();
+                    newTSTest.Runs.Add(newtsRun);
+                }
+            }catch(Exception ex)
+            {
+                Reporter.ToLog(eLogLevel.ERROR, "Failed to pull QC test case RUN info", ex);
+                newTSTest.Runs = new List<QtestTestRun>();
+            }
+
+            return newTSTest;
+        }
+
+        private static List<QtestTestRun> GetListTSTestRuns(QtestTest tsTest)
+        {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)